data/human/outfits.txt

```diff
@@ -251,9 +251,9 @@ outfit "Surveillance Pod"
 	"mass" 5
 	"outfit space" -5
 	"outfit scan power" 16
-	"outfit scan speed" 2
+	"outfit scan efficiency" 20
 	"cargo scan power" 4
-	"cargo scan speed" 2
+	"cargo scan efficiency" 10
```

Comment:
If I'm reading this correctly, this is buffing the Surveillance pod, correct? 

Just noting that everywhere else translates the exact same value from scan speed directly into scan efficiency, but the surveillance pod and the Kas'ik Tek 7 are the two exceptions where the value has been increased.

Reply:
Most outfits had `"(cargo|outfit) scan speed" 1`, which meant a 60 frame scan time. (A scan speed of 2 equals about 43 frames, AKA 60 / sqrt(2) frames.) A decent translation is making scan efficiency to be somewhere around 15x original value, or set efficiency to speed * power.

So, this isn't as much a buff as it is "accounting for inflation" because the scan speed was 2 rather than 1, it warrants a higher scan efficiency